tests/csrc/unittests/test_sampling_layer.cu

```diff
@@ -1260,6 +1261,7 @@ TYPED_TEST(SamplingDecodeTest2, CorrectnessBatchRandTopP)
     this->runCurandTest({113, 1201, 1, 0, 1.0f, 5}, false, false);
 }
 
+#if 0
```

Comment:
We still need to test topk sampling

Reply:
The dynamic decoder doesn't support process mini-batch anymore